10-JavaScript/dom-manipulation-live-code-notes.md

```diff
@@ -0,0 +1,136 @@
+# DOM Manipulation the Old Way: Live-code Notes
+
+## Running JavaScript on Websites
+
+Obviously we don't expect users to type out all their own JavaScript by hand. How do we get a website to connect to a JavaScript file? We'll add a link to our JavaScript file in our HTML, similar to the way we included CSS before.
+
+When our user goes to a website using their browser, the browser will make a request for an HTML page. This HTML page contains references to our CSS and JavaScript files. Then, these files will be loaded into the browser.
+
+1. Create a new directory called `browser-js`, with two files: `index.html` and `index.js`:
+
+1. Add this basic HTML to `index.html`:
+
+    ```html
+    <!-- index.html -->
+    <!DOCTYPE html>
+    <html>
+      <head>
+        <meta charset="utf-8">
+        <title>JavaScript Test Page</title>
+      </head>
+      <body>
+        <h1>Test page for JavaScript in the Browser</h1>
+
+        <div id="js-lecture-target">This is text that lives inside a div element with an id "js-lecture-target"!</div>
+      </body>
+    </html>
+    ```
+
+1. And finally, add this JavaScript to `index.js`:
+
+    ```javascript
+    // index.js
+    console.log('This is a test');
+    ```
+
+1. Modify your HTML page `index.html` by adding this `<script>` tag at the end of `<body>`, RIGHT before the closing `</body>` tag:
```

Comment:
Add a note about needing the `</script>` tag since `<script>` isn't self closing?